dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -617,6 +622,10 @@ private WorkflowVersion setupServiceFilesForGitHubVersion(Triple<String, Date, S
                 file.setPath(filePath);
                 file.setContent(fileContent);
                 file.setType(DescriptorLanguage.FileType.DOCKSTORE_SERVICE_OTHER);
+                Optional<String> sha = FileFormatHelper.calcSHA1(file.getContent());
```

Comment:
This is fine, but FYI, you could write it to be something like this (didn't try to compile it, so syntax could be off), which is my own personal preference:

```
FileFormatHelper.calcSHA1(file.getContent()).ifPresent(sha -> file.getChecksums().add(new Checksum(SHA_TYPE_FOR_SOURCEFILES, sha));
```